providers/util.go

```diff
@@ -0,0 +1,31 @@
+package providers
+
+import (
+	"fmt"
+	"net/http"
+)
+
+const (
+	tokenTypeBearer = "Bearer"
+	tokenTypeToken  = "token"
+
+	acceptHeader          = "Accept"
+	acceptApplicationJSON = "application/json"
+)
+
+func getAuthorizationHeader(prefix, token string, extraHeaders map[string]string) http.Header {
+	header := make(http.Header)
+	for key, value := range extraHeaders {
+		header.Add(key, value)
+	}
+	header.Set("Authorization", fmt.Sprintf("%s %s", prefix, token))
+	return header
+}
+
+func getOIDCHeader(accessToken string) http.Header {
```

Comment:
NIT: Since these aren't traditional getters and are more builder methods, should the verb prefix for these method names be `make` or `build` instead of `get`?

Reply:
I like `makeOIDCHeader`, I'll rename!